pkg/security/tests/network_device_test.go

```diff
@@ -133,3 +138,157 @@ func TestNetDevice(t *testing.T) {
 		}
 	})
 }
+
+func TestTCFilters(t *testing.T) {
+	checkKernelCompatibility(t, "RHEL, SLES and Oracle kernels", func(kv *kernel.Version) bool {
+		// TODO: Oracle because we are missing offsets
+		return kv.IsRH7Kernel() || kv.IsOracleUEKKernel() || kv.IsSLESKernel()
```

Comment:
So what is the behavior on Oracle? Also I'm wondering if we can make this a util function, since I'm running into a similar issue with Event Monitor, where the check is in the Probe, but the unit test doesn't have the probe.

Reply:
I'm not sure about the behavior on Oracle, I just copied the compatibility check from the other network device test.
I agree this could be a util function as this check is performed for every network-related test.
If the check is in the Probe you can probably call it through the `testModule` struct in the unit test.